site/src/pages/components/selectlist.mdx

```diff
@@ -397,3 +397,4 @@ Given that we would be required to introduce something new into the platform (at
 - [Why not use the `slot` or `behavior` attributes](https://github.com/openui/open-ui/issues/702)
 - [Why not support the `size` attribute](https://github.com/openui/open-ui/issues/153)
 - [Why support `:checked` instead of `:selected`](https://github.com/openui/open-ui/issues/827)
+- [Why allow interactive content in `<listbox>`](https://github.com/openui/open-ui/issues/540)
```

Comment:
nit: `Allow interactive content in <listbox> but with strong warnings`

Reply:
done